unicodetools/data/ucd/dev/DerivedCoreProperties.txt

```diff
@@ -1371,6 +1371,7 @@ FFDA..FFDC    ; Alphabetic # Lo   [3] HALFWIDTH HANGUL LETTER EU..HALFWIDTH HANG
 1DF0A         ; Alphabetic # Lo       LATIN LETTER RETROFLEX CLICK WITH RETROFLEX HOOK
 1DF0B..1DF1E  ; Alphabetic # L&  [20] LATIN SMALL LETTER ESH WITH DOUBLE BAR..LATIN SMALL LETTER S WITH CURL
 1DF25..1DF2A  ; Alphabetic # L&   [6] LATIN SMALL LETTER D WITH MID-HEIGHT LEFT HOOK..LATIN SMALL LETTER T WITH MID-HEIGHT LEFT HOOK
+1DF2D..1DF3A  ; Alphabetic # L&  [14] LATIN SMALL LETTER D WITH HOOK AND PALATAL HOOK..LATIN LETTER PHARYNGEAL VOICED FRICATIVE WITH PALATAL HOOK
```

Comment:
I find it curious that even a range consisting of only `Ll` gets `L&` but this is by no means new.

Reply:
Yes, this makes me sad. The Kenfiles (EastAsianWidth, LineBreak, VerticalOrientation) do that right, I wrote some code for that when I made the tools capable of regenerating them; but the derived files were doing it in a simpler and uglier way.

Reply:
I figured as much. This was really just a drive-by observation and was not meant to be part of my review.